lib/vagrant-libvirt/action/prepare_nfs_settings.rb

```diff
@@ -59,7 +59,7 @@ def read_machine_ip(machine)
           return ssh_host if ping(ssh_host)
 
           # check other ips
-          command = "ip addr show | grep -i 'inet ' | grep -v '127.0.0.1' | tr -s ' ' | cut -d' ' -f3 | cut -d'/' -f 1"
+          command = "/sbin/ip addr show | grep -i 'inet ' | grep -v '127.0.0.1' | tr -s ' ' | cut -d' ' -f3 | cut -d'/' -f 1"
```

Comment:
I just did a test with `$(which ip)` instead of just `ip`, but as expected it fails because /sbin isn't in the default PATH of my vagrant guest machine (RHEL6).

@pronix May I ask what Linux distro you're currently using that has the `ip` tool in `/usr/sbin` and not in `/sbin`? I've done verifications on Fedora 25, RHEL6 and RHEL7 and all of them provide `/sbin/ip`.

In the case of Fedora 25 and RHEL7 the executable itself is actually at `/usr/sbin/ip` (the RPM package `iproute` owns this), but on those distro `/sbin` is a symlink to `/usr/sbin` so both `/sbin/ip` and `/usr/sbin/ip` work there.

I also looked at the upstream `iproute2` build system (http://git.kernel.org/cgit/linux/kernel/git/shemminger/iproute2.git/tree/Makefile and http://git.kernel.org/cgit/linux/kernel/git/shemminger/iproute2.git/tree/ip/Makefile) and that defaults installing the `ip` tool to `/sbin` as well so I think it's pretty safe to assume that `/sbin/ip` is also valid for other distros.

If you think the risk of unexpected breakage is still too big with the patch as is I can also implement the suggestion which was done by @electrofelix as it provides a nice fallback which covers both cases.